panel/dist/css/chat_message.css

```diff
@@ -9,7 +9,6 @@
 
 :host {
   max-height: none;
-  max-width: 99%; /* Prevent horizontal scroll bar */
```

Comment:
Slightly worried that this is important to prevent it from creating an ugly horizontal scroll bar.

Reply:
Yeah text isn't the issue with the max_width on the text but I see what you mean. I just don't think this can be on `:host` since you apply this stylesheet to a bunch of things. It's fine for a max-width of 99% on the message itself.